api/db-cdr/generate-cdr/build-cloudsql-tables.sh

```diff
@@ -109,7 +109,8 @@ VALUES
 (3,'Domains',4,'Labs and Measurements','Labs and Measurements',0,0,1,3),
 (4,'Domains',5,'Observations','Observations',0,0,1,4),
 (5,'Domains',6,'Procedures','Procedures',0,0,1,5),
-(6,'Program Physical Measurements',19,'Physical Measurements','Participants have the option to provide a standard set of physical measurements as part of the enrollment process(program physical measurements)',0,0,0,8)"
+(6,'Program Physical Measurements',19,'Physical Measurements','Participants have the option to provide a standard set of physical measurements as part of the enrollment process(program physical measurements)',0,0,0,8),
```

Comment:
You also need to add code to update devices count. Example for updating counts for standard condition occurrence: https://github.com/all-of-us/workbench/blob/013e4226558dbc16de7a894d5d0820e08fc7efa5/api/db-cdr/generate-cdr/build-cloudsql-tables.sh#L248-L257

Reply:
got it. done